main.go

```diff
@@ -89,10 +90,12 @@ func main() {
 		certSigningDisabled               bool
 		certManagerEnabled                bool
 		maxKafkaTopicConcurrentReconciles int
+		healthProbeAddr                   string
```

Comment:
**Question**: Is there a particular reason we put this field here? :)

Reply:
I placed this variable on the current position because I recognized there is no alphabetical order and that's why I chose historical order.